include/class.dynamic_forms.php

```diff
@@ -1282,7 +1282,14 @@ function saveAnswers($isEditable=null, $refetch=false) {
 
             try {
                 $field->setForm($this);
-                $val = $field->to_database($field->getClean());
+                if (get_class($field) == 'FileUploadField') {
```

Comment:
Perhaps a few comments here would be ideal to explain why the logic is needed here?